docs/source/examples/managed-jobs.rst

```diff
@@ -159,16 +115,89 @@ We can launch it with the following:
 SkyPilot will launch and start monitoring the job. When a spot preemption or any machine failure happens, SkyPilot will automatically
 search for resources across regions and clouds to re-launch the job.
 
-In this example, the job will be restarted from scratch after each preemption recovery.
-To resume the job from previous states, user's application needs to implement checkpointing and recovery.
+
+.. _spot-jobs:
+
+Running on Spot
+---------------
+
+Managed jobs can run on spot instances, and preemptions are auto-recovered by SkyPilot.
+
+.. tip::
+   Spot instances are cloud VMs that may be "preempted".
+   The cloud provider can forcibly shut down the underlying VM and remove your access to it, interrupting the job running on that instance.
+   This can be inconvenient, but SkyPilot will automatically restart/recover your job on another instance.
+
+   In exchange for unpredictability, spot instances are significantly cheaper than normal instances that are not subject to preemption (so-called "on-demand" instances).
+   Depending on the cloud and VM type, spot instances can be 70-90% cheaper.
+   Since SkyPilot automatically handles preemptions, you can get these savings for free.
+
+To run on spot instances, use :code:`sky jobs launch --use-spot`, or specify :code:`use_spot: true` in your SkyPilot YAML.
```

Comment:
Pull this up before the tip above?

Reply:
Also, this para feels long; is there a way you see to break it up / simplify a little?